ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/ConnectorType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.api.client;
+
+public interface ConnectorType {
+  enum Type { SOURCE, SINK, UNKNOWN }
```

Comment:
We should add a javadoc about what situations might cause the connector type to be `UNKNOWN`, since I imagine will be confused about this otherwise.